.github/CHANGELOG.md

```diff
@@ -216,142 +203,41 @@
-
-@qml.qnode(dev)
-def kernel_circuit(x1, x2):
-    qml.templates.AngleEmbedding(x1, wires=wires)
-    qml.adjoint(qml.templates.AngleEmbedding)(x2, wires=wires)
-    return qml.probs(wires)
-
-kernel = lambda x1, x2: kernel_circuit(x1, x2)[0]
-
-# "Training feature vectors"
-X_train = np.random.random((10, 6))
-# Create symmetric square kernel matrix (for training)
-K = qml.kernels.square_kernel_matrix(X_train, kernel)
-# Add some (symmetric) Gaussian noise to the kernel matrix.
-N = np.random.randn(10, 10)
-K += (N + N.T) / 2
-
-K1 = qml.kernels.displace_matrix(K)
-K2 = qml.kernels.closest_psd_matrix(K)
-K3 = qml.kernels.threshold_matrix(K)
-K4 = qml.kernels.mitigate_depolarizing_noise(K, num_wires, method='single')
-K5 = qml.kernels.mitigate_depolarizing_noise(K, num_wires, method='average')
-K6 = qml.kernels.mitigate_depolarizing_noise(K, num_wires, method='split_channel')
-
-# "Testing feature vectors"
-X_test = np.random.random((5, 6))
-# Compute kernel between test and training data.
-K_test = qml.kernels.kernel_matrix(X_train, X_test, kernel)
-```
-
-* QNodes now display readable information when in interactive environments or when printed.
-  [(#1359)](https://github.com/PennyLaneAI/pennylane/pull/1359).
-
-* Added CPhase operation as an alias for ControlledPhaseShift operation
-  [(#1319)](https://github.com/PennyLaneAI/pennylane/pull/1319).
-
-* The `qml.Toffoli` operation now has a decomposition over elementary gates.
-  [(#1320)](https://github.com/PennyLaneAI/pennylane/pull/1320)
+* Adds a `hamiltonian_expand` tape transform. This takes a tape ending in
+  `qml.expval(H)`, where `H` is a Hamiltonian, and maps it to a collection
+  of tapes which can be executed and passed into a post-processing function yielding
+  the expectation value.
+  [(#1142)](https://github.com/PennyLaneAI/pennylane/pull/1142)
 
-* Added a new noise channel, `qml.ResetError`.
-  [(#1321)](https://github.com/PennyLaneAI/pennylane/pull/1321).
+  Example use:
 
-* The `qml.SWAP`  operation now has a decomposition over elementary gates. [(#1329)](https://github.com/PennyLaneAI/pennylane/pull/1329)
+  ```python
+  H = qml.PauliZ(0) + 3 * qml.PauliZ(0) @ qml.PauliX(1)
 
-* Added functionality for constructing and manipulating the Pauli group
-  [(#1181)](https://github.com/PennyLaneAI/pennylane/pull/1181).
-  The function `pennylane.grouping.pauli_group` provides a generator to
-  easily loop over the group, or construct and store it in its entirety.
-  For example, we can construct the 3-qubit Pauli group like so:
+  with qml.tape.QuantumTape() as tape:
+      qml.Hadamard(wires=1)
+      qml.expval(H)
 
-  ```pycon
-  >>> from pennylane.grouping import pauli_group
-  >>> pauli_group_1_qubit = list(pauli_group(1))
-  >>> pauli_group_1_qubit
-  [Identity(wires=[0]), PauliZ(wires=[0]), PauliX(wires=[0]), PauliY(wires=[0])]
+  tapes, fn = qml.transforms.hamiltonian_expand(tape)
+  dev = qml.device("default.qubit", wires=3)
+  res = dev.batch_execute(tapes)
```

Comment:
I think this split here between python/pycon occurs more naturally after the tape context

Reply:
```python
H = qml.PauliZ(0) + 3 * qml.PauliZ(0) @ qml.PauliX(1)

with qml.tape.QuantumTape() as tape:
    qml.Hadamard(wires=1)
    qml.expval(H)

tapes, fn = qml.transforms.hamiltonian_expand(tape)
```

We can now evaluate the transformed tapes, and apply
the post-processing function:
```pycon
>>> dev = qml.device("default.qubit", wires=3)
>>> res = dev.batch_execute(tapes)
>>> fn(res)
3.999999999999999
```